BlazarService/src/test/java/com/hubspot/blazar/service/DependencyVersionTest.java

```diff
@@ -43,11 +45,10 @@
   @Inject
   private DependenciesService dependenciesService;
 
+  @Inject
   @Before
-  public void before() throws Exception {
-    // set up the data for these inter-project build tests
-    // todo migrate inter project data to a more common place
-    runSql("InterProjectData.sql");
+  public void before(ManagedDataSource dataSource) throws Exception {
+    runSql(dataSource, "InterProjectData.sql");
```

Comment:
I should rename this migration, but its just test data, initially added for inter project builds, but now used as a starting point for most service layer tests.